layouts/shortcodes/tooltip.html

```diff
@@ -5,6 +5,7 @@
     "commP" "The commitment phase of the Proof-of-Replication (PoRep) process."
     "cryptographic proving systems" "A cryptographic protocol where a prover proves something to a verifier. Used by all blockchains."
     "consensus" "Any method used to achieve agreement, trust, and security in a blockchain. "
+    "deferred execution" "A mechanism by which the execution of every message is deferred by one epoch."
```

Comment:
Not actually sure if this is a "rule", but I feel like this reads a little smoother. Might just be my writing style

```suggestion
    "deferred execution" "A mechanism in which the execution of every message is deferred by one epoch."
```